### PR TITLE
Basic testing setup

### LINKED ISSUE #12: Should have login tests

### PR DESCRIPTION
**Pull request by kundanvishen**
This PR resolves #12

### CODE REVIEW

**Comment by andrewlinfoot** [comment]
Are you planning on using graphql?

**Comment by kundanvishen** [comment]
@andrewlinfoot Yes, we are planning on using GraphQL, learning/reading docs it on the side.

File: __tests__/server.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+describe('index route', () => {
+  afterEach(() => {
+    app.server.close();
+  });
+
+  test('should respond with a 200 with no query parameters', () => {
+    return request(app)
+      .get('/')
+      // .expect('Content-Type', /html/)
```
Comment (ArfatSalman) [inline __tests__/server.test.js]: No spurious comments.

File: src/server.js [hunk 2]
```diff
@@ -1,9 +1,19 @@
 require('dotenv').config();
+
 const express = require('express');
+const cors = require('cors');
 
 const PORT = process.env.PORT || 3000;
```
Comment (ArfatSalman) [inline src/server.js]: Use a constants file to have a central repository of exported constants. 

Read this. [https://github.com/nodejs/node/issues/3104](url)

**Comment by kundanvishen** [comment]
@ArfatSalman Thank you for the review. I have made the suggested changes. Kindly review.

**Pull request merged**